Dockerfile

```diff
@@ -716,6 +722,7 @@ RUN <<END
     ln -s /etc/ssl /rootfs/usr/local/share/ca-certificates
     ln -s /etc/ssl /rootfs/etc/ca-certificates
 END
+RUN mkdir -p /rootfs/selinux
```

Comment:
let's unify this with other mkdirs, see line 703-707

Reply:
Will remove as we mount nothing here now.